### PR TITLE
Fixes 176: add list sorting

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
* [ ] need to update/add tests

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-176

File: src/services/Content/ContentQueries.ts [hunk 1]
```diff
@@ -20,10 +20,15 @@ export const CONTENT_LIST_KEY = 'CONTENT_LIST_KEY';
...
+  page: number,
+  limit: number,
+  filterData: FilterData,
+  sortBy: string,
+) =>
   useQuery<ContentListResponse>(
-    [CONTENT_LIST_KEY, page, limit, ...Object.values(filterData)],
-    () => getContentList(page, limit, filterData),
+    [CONTENT_LIST_KEY, page, limit, sortBy, ...Object.values(filterData)],
+    () => getContentList(page, limit, filterData, sortBy),
```
Comment (Andrewgdewar) [inline src/services/Content/ContentQueries.ts:31]: You need to add "sortBy" to the delete function to keep our optimistic updates happening. If you slow down the network tab you'll see what I mean after deleting an item.

Comment (jlsherrill) [reply src/services/Content/ContentQueries.ts:31]: i didn't actually see much of a problem when running on slow 3g, but i went ahead and updated what i think you mean.  let me know!

File: src/components/ContentListTable/ContentListTable.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ import {
   Td,
   Th,
   Thead,
+  ThProps,
```
Comment (Andrewgdewar) [inline src/components/ContentListTable/ContentListTable.tsx:21]: Good use of this. It's stuff like this that make me feel Golang pairs well with typescript.

Comment (Andrewgdewar) [reply src/components/ContentListTable/ContentListTable.tsx:21]: >> ThProps['sort']

**Review by Andrewgdewar — APPROVED** [review]

**Comment by Andrewgdewar** [comment]
I'm not sure there is much to test here to be honest. If you see a test that will be meaningful do go ahead.

**Comment by Andrewgdewar** [comment]
You may want to rebase from [ticket 158](https://github.com/content-services/content-sources-frontend/pull/34), as it removes the url column.

**Comment by jlsherrill** [comment]
updated, but my local environment isn't working for some reason.... so i can't test the result

**Comment by jlsherrill** [comment]
alright, good to go

**Pull request merged**